pages/services/prometheus/0.1.0-2.3.1/release-notes/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes
+title: Release Notes
+menuWeight: 130
+excerpt: Release Notes for version 0.1.0-2.3.1
+featureMaturity:
+enterprise: false
+---
+## Version 2.3.1
+
+This is the first release of Prometheus to Universe. Based on the latest stable release version of Prometheus, version 2.3.1, this installation would be supported on DCOS cluster 1.10 and above. This has been built using current stable version of SDK (Version 0.42.1).
```

Comment:
Remove `This has been built using current stable version of SDK (Version 0.42.1).`

Reply:
Done